docs/source/examples/frameworks/gke_dws.md

```diff
@@ -0,0 +1 @@
+../../generated-examples/gke_dws.md
```

Comment:
We need add the new configs to config.rst

Reply:
Updated